package.json

```diff
@@ -2,14 +2,16 @@
   "name": "dietapp",
   "version": "0.0.7",
   "description": "Sistema para la creación de dietas personalizadas",
+  "main": "src/index.js",
   "scripts": {
     "check": "tsc --noEmit",
     "build": "tsc --build",
     "clean": "tsc --build --clean",
     "postclean": "rm -rf dist && rm -rf log*",
     "test": "jest --detectOpenHandles",
```

Comment:
Vale, ya estoy viendo por qué pones esto. Tratad de haced los tests bien y manejar bien código asíncrono, no hagáis esta ñapa que lo único que hace es salir del test sin esperar a que se resuelvan las promesas.